src/DTS/Render/Render.php

```diff
@@ -0,0 +1,25 @@
+<?php declare(strict_types=1);
+
+
+namespace HomeCEU\DTS\Render;
+
+
+abstract class Render implements RenderInterface {
```

Comment:
any particular reason for this to be abstract if you don't have abstract methods?

Reply:
I didn't want to allow use of this class, I wanted to the user to use the factory to create a Renderer, and I'd consider it "abstract" in the sense it implements an interface and in order to extend the class, you must implement those methods.